pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -677,6 +677,18 @@ class Timestamp(_Timestamp):
         """
         return cls(datetime.fromtimestamp(ts))
 
+    # Issue 25016. As strptime with %z is not supported in Python 2.
```

Comment:
We're dropping Python 2 in the next release, so I don't think this comment is needed.